source/mainnet/net/resources/ccd-scan-transactions.rst

```diff
@@ -0,0 +1,61 @@
+.. _transactions-view:
+
+=============================
+CCDScan Transactions overview
+=============================
+
+The Transactions overview shows information about the transactions during the selected time range in the filter.
+
+.. image:: ../images/ccd-scan/ccd-scan-transactions.png
+
+|
+
+The graphs show the number of **cumulative transactions** since genesis and the number of **Transactions** in a range as determined by the time selected in the filter.
+
+Below the graphs, the table contains the following:
+
+- **Transaction hash**: the first six characters of the transaction hash. You can click Copy |copy| to copy the entire hash. Click the transaction hash to see the :ref:`transaction details<home-screen-transaction>`.
+- **Status**: transaction status
+- **Age**: transaction age
+- **Type**: transaction type
+- **Block height**: the block height of the transaction.
+- **Sender**: the first six characters of the account that sent the transaction. You can click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
+- **Cost**: the transaction fee.
+
+.. _home-screen-transaction:
+
+Transaction details
+===================
+
+When you click a transaction hash, transaction information appears. The information is different depending upon whether the transaction was successful or rejected.
+
+.. image:: ../images/ccd-scan/ccd-scan-home-transaction-success.png
+
+|
+
+The transaction details shows the following.
+
+- **Transaction**: the first six characters of the transaction hash. Click Copy |copy| to copy the entire transaction hash. It also shows the transaction state.
+- **Block height/block hash**: the block height and the first six characters of the block hash. Click Copy |copy| to copy the entire block hash. Click the block hash to see the :ref:`account details<blocks-view>`.
+- **Age**: the transaction age with date/time stamp.
+- **Transaction type/cost**: the transaction type and the transaction fee.
+- **Sender**: the first six characters of the account that sent the transaction if available for that transaction. Click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
+- **Events/Reject reason**:  list of events included in the transaction. It contains more details about the transaction, including drill-through links and copy options where available. There are many different transaction types and descriptions.
+
+A successful transaction shows the **Events** that were included in the transaction.
+
+If the transaction was rejected, you see something similar to the following:
+
+.. image:: ../images/ccd-scan/ccd-scan-home-transaction-reject.png
+
+|
+
+The only difference between a successful and rejected transaction is that the rejected transaction shows the **Reject reason** instead of the events.
```

Comment:
Not sure what the context for "the only difference" is.

But I would not phrase it like this. Instead I would say something like

"The only effect of a rejected transaction is the payment for execution cost. In contrast, a successful transaction affects the state of the chain. The effects are described by the generated events."